.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
```

Comment:
It excludes temporary saves from some text editors. Never heard of someone having a problem with this rule in production. Never saw `filename~` used for anything else than temporary files.